src/utils/project-content/get-project-content.ts

```diff
@@ -87,15 +88,26 @@ function _getCollectionFragments(collectionDirectory: string): IFragment[] {
       }
     })
     .map((directory) => {
+      let unknownFiles = glob
+        .sync(path.join(directory, '*'), { dot: true })
+        .map((filePath) => path.relative(directory, filePath))
+        .filter((filePath) => filePath !== 'fragment.json');
+
       const metadata = _readJSONSync<IFragmentMetadata>(
         path.resolve(directory, 'fragment.json')
       );
 
       const readFile = (
         filePath: string,
-        encoding: 'utf-8' | undefined
+        encoding: 'utf-8' | undefined = undefined
       ): string | Buffer => {
         try {
+          const [baseDir] = filePath.split(path.sep);
+
+          unknownFiles = unknownFiles.filter(
+            (_filePath) => _filePath !== filePath && _filePath !== baseDir
+          );
+
```

Comment:
Wouldn't it be better to do this filtering explicitly instead of hiding it in the readFile function?

With this, I mean to filter the unknowFiles doing the filtering after obtaining all the know files

```
js
const savedPaths = [metadata.htmlPath, metadata.cssPath, ...]

unknowFiles = unknowFiles.filter(_filePath => !savedPaths.includes(_filePath))
```

Reply:
:+1: yep, that sounds much better :smile: